packages/flutter/lib/src/cupertino/dialog.dart

```diff
@@ -423,20 +424,30 @@ class _CupertinoAlertDialogState extends State<CupertinoAlertDialog> {
   }
 }
 
-/// Rounded rectangle surface that looks like an iOS popup surface, e.g., alert dialog
-/// and action sheet.
+/// An iOS-style rounded rectangle surface useful for showing modal overlays,
+/// such as dialogs and action sheets.
 ///
-/// A [CupertinoPopupSurface] can be configured to paint or not paint a white
-/// color on top of its blurred area. Typical usage should paint white on top
-/// of the blur. However, the white paint can be disabled for the purpose of
-/// rendering divider gaps for a more complicated layout, e.g., [CupertinoAlertDialog].
-/// Additionally, the white paint can be disabled to render a blurred rounded
-/// rectangle without any color (similar to iOS's volume control popup).
+/// By default, this surface is composed of a transparent surface color painted
+/// atop a [BackdropFilter]. The surface color is light gray when the ambient
+/// [CupertinoTheme] brightness is [Brightness.light], and dark gray when the
+/// ambient brightness is [Brightness.dark]. The [BackdropFilter] applied
+/// beneath the surface saturates and blurs underlying content.
+///
+/// The blur strength can be changed by setting [blurSigma] to a positive value,
+/// or removed by setting the [blurSigma] to 0.
+///
+/// The saturation effect can be removed by setting [isVibrancePainted] to
+/// false. Removing the vibrance can prevent oversaturating areas where multiple
+/// [CupertinoPopupSurface]s are stacked upon eachother.
```

Comment:
I suppose this is added actually for the only purpose of testing, right?

Reply:
Stacked colorfilters did cause some unpleasant oversaturation on the nested CupertinoMenuAnchor I was working on, to the point where I actually disabled color filters on nested layers. But to your point, this is primarily for testing convenience. Otherwise, it's could be helpful to users who want consistent appearance across web and mobile. Should I just remove the explanation or call it "debugIsVibranceEnabled"?